pkg/core/bundle_unpack.go

```diff
@@ -18,15 +20,60 @@ const (
 	fileDownloadsPerConcurrentChunks = 3
 )
 
-func unpackBundleDescriptor(ctx context.Context, bundle *Bundle) error {
-
-	bundleDescriptorBuffer, err := storage.ReadTee(ctx,
-		bundle.MetaStore, model.GetArchivePathToBundle(bundle.RepoID, bundle.BundleID),
-		bundle.ConsumableStore, model.GetConsumablePathToBundle(bundle.BundleID))
+func setBundleIDFromConsumableStore(ctx context.Context, bundle *Bundle) error {
```

Comment:
Why not getBundleIDFromPath?

Reply:
i've added a `getBundleIDFromPath` function that returns a bundle id given the path to a bundle descriptor within the bundle (and the empty string when given another path such as a filelist path) and used it within `setBundleIDFromConsumableStore`.

it occurs to me that what `getBundleIDFromPath` could also mean is to get the bundle id from a path to a bundle on the local filesystem, except i suppose keeping the abstraction away from the local filesystem via `storage` is a desirable abstraction.

also, i could inline the contents of `setBundleIDFromConsumableStore` since it's only used once .. or not:  i'm ambivalent re. the aesthetics of whether to maintain `setBundleIDFromConsumableStore` as a separate function or not.